src/TeleportJoin.sol

```diff
@@ -60,6 +62,7 @@ contract TeleportJoin {
     bytes32     immutable public ilk;
     bytes32     immutable public domain;
 
+    string  constant public ARITHMETIC_ERROR = string(abi.encodeWithSignature("Panic(uint256)", 0x11));
```

Comment:
I don't think this needs to be `public`. Really I don't think any of these constants need to be public as they have nothing to do with the external interface.